src/index.html

```diff
@@ -13,6 +13,38 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length-31 calendar--start-day-sun">
```

Comment:
It seems like you're manually creating each day of the calendar, which can be very repetitive and error-prone. Consider using a scripting language like JavaScript to dynamically generate these days based on the month length and start day. This will make your code more maintainable and scalable.